client/src/components/Workflow/Invocations.vue

```diff
@@ -65,7 +65,10 @@
                 <UtcDate :date="data.value" mode="elapsed" />
             </template>
             <template v-slot:cell(execute)="data">
-                <WorkflowRunButton :id="getWorkflowByInstanceId(data.item.workflow_id).id" :root="root" />
+                <WorkflowRunButton
+                    v-if="getWorkflowByInstanceId(data.item.workflow_id)"
```

Comment:
This isn't the only issue here, should I pull out 
Would this fix it https://github.com/galaxyproject/galaxy/pull/14067/commits/8be0f90714fa2edcb893ada7b0604ed681e6de07 too ? ?

Reply:
That is a better fix than mine!